src/themes/saas/space.tsx

```diff
@@ -0,0 +1,64 @@
+export const spacing = {
```

Comment:
sorry, this is needed to be specified? I don't know if could works with chakra defaults?

Reply:
After a talk with Gerard i knew why is used